tests/perf-test/OptionsSelector.perf-test.tsx

```diff
@@ -35,8 +38,11 @@ jest.mock('../../src/components/withNavigationFocus', () => (Component) => {
     return WithNavigationFocus;
 });
 
-const generateSections = (sectionConfigs) =>
-    _.map(sectionConfigs, ({numItems, indexOffset, shouldShow = true}) => ({
+type GenerateSectionsProps = Array<{numItems: number; indexOffset: number; shouldShow?: boolean}>;
+
+const generateSections = (sectionConfigs: GenerateSectionsProps) =>
+    sectionConfigs.map(({numItems, indexOffset, shouldShow = true}) => ({
+        // eslint-disable-next-line @typescript-eslint/naming-convention
         data: Array.from({length: numItems}, (_v, i) => ({
```

Comment:
NAB and I see `_v` isn't used so I imagine the `_` prefix is intentional, but do we have this in our style guide, or can we use a regular name for this variable?
```suggestion
        data: Array.from({length: numItems}, (v, i) => ({
```